src/components/User/UserTrainingModels.tsx

```diff
@@ -138,8 +178,21 @@ export default function UserTrainingModels() {
     },
   });
 
-  // TODO [bw]: probably need to do something if attempting to delete while running
-  const handleDeleteModel = (model: (typeof items)[number]) => {
+  const goToModel = (e: React.MouseEvent<HTMLTableRowElement>, href: string) => {
+    if (opened) return false;
+    // on control click or middle click, open in new tab
+    if ((e.ctrlKey && e.button === 0) || e.button === 1) {
+      e.preventDefault();
+      window.open(href, '_blank');
+    } else if (e.button === 0) {
+      router.push(href);
+    }
+  };
+
+  const handleDeleteModel = (e: React.MouseEvent<HTMLButtonElement>, model: any[][number]) => {
```

Comment:
I thiiiink we can get the type here as well
```suggestion
  const handleDeleteModel = (e: React.MouseEvent<HTMLButtonElement>, model: ModelGetMyTrainingModels[number]) => {
```
Here's another approach
```suggestion
  const handleDeleteModel = (e: React.MouseEvent<HTMLButtonElement>, model: (typeof items)[number]) => {
```

Reply:
did this was `MyTrainingModelGetAll['items'][number]`